### ISSUE TITLE
VSTS-CI uses incorrect Gson jar version in Fortify SCA phase

### ISSUE THREAD

**Issue by iambmelt** [AzureAD/azure-activedirectory-library-for-android#914]
Discovered during #897 

Fortify SCA uses a `C://FortifyJarDependency` path to locate & scan 3rd party libraries -- the version being scanned by Fortify (`2.2.4`) does not match the version we ship with ADAL (`2.8.1` as of 7/24/2017).

The `.jar` located at `C://FortifyJarDependency` should be updated to match the version we ship in `versions.gradle`

**Comment by iambmelt** [comment]
Resolved in build [20170724.4](https://identitydivision.visualstudio.com/IDDP/_build/index?buildId=12749&_a=summary)

**Comment by iambmelt** [comment]
Updated script to resolve correct jar through build environment variable. This change closes #897 and #914
```sh
\bin\sourceanalyzer.exe-b msalandroidbuild .
-exclude ./**/androidTest/**/*.java
-exclude ./**/test/**/*.java 
-exclude ./**/automationtestapp/**/R.java
-exclude ./**/adal/**/R.java
-exclude ./**/R.java
-exclude ./**/BuildConfig.java
-exclude ./**/IBrokerAccountService.java
-cp "C:\AndroidSDK\platforms\android-$(android:apiVersion)\android.jar"
-cp "C:\FortifyJarDependency\gson-$(android:gsonVersion).jar"
-cp "C:\FortifyJarDependency\android-support-customtabs.jar"
-cp "C:\AndroidSDK\platforms\android-$(android:apiVersion)\optional\org.apache.http.legacy.jar"
-jdk 1.8
```
